feat(curriculum): dice game project (#50819)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649a78fd5dd71202affcffcb.md

---
id: 649a78fd5dd71202affcffcb
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

> You need to create a function named `rollDice` that will be used to roll the dice and update the dice display on the result screen.

Reviewer: ```suggestion
Use the `function` keyword to declare a function named `rollDice`. This will be used to roll the dice and update the dice display on the result screen.
```

# --hints--

> You should create a function named `rollDice`

Reviewer: ```suggestion
You should use the `function` keyword to declare `rollDice`.
```

```js

> assert.match(code, /function\s*rollDice\(\)/);

Reviewer: ```suggestion
assert.match(code, /function\s*rollDice\s*\(\s*\)/);
```

```

`rollDice` should be an empty function.

```js

> assert.match(code, /function\s*rollDice\s*\(\)\s*{\s*}/);

Reviewer: ```suggestion
assert.match(code, /function\s*rollDice\s*\(\s*\)\s*{\s*}/);
```
